plugins/techdocs/src/home/components/TechDocsCustomHome.tsx

```diff
@@ -36,6 +40,7 @@ import {
 } from '@backstage/core';
 import { DocsTable } from './DocsTable';
 import { DocsCardGrid } from './DocsCardGrid';
+import { useOwnUser } from '../hooks';
```

Comment:
should this reuse the existing hook from `@backstage/plugin-catalog-react`? 
https://github.com/backstage/backstage/blob/2147ab9c1bd37e93e9e332c5ee14daa08be031e6/plugins/catalog-react/src/hooks/useOwnUser.ts#L26

Reply:
> should this reuse the existing hook from `@backstage/plugin-catalog-react`?
> https://github.com/backstage/backstage/blob/2147ab9c1bd37e93e9e332c5ee14daa08be031e6/plugins/catalog-react/src/hooks/useOwnUser.ts#L26

Yes would love to now when its moved over to the catalog-react plugin, when we [copied this over last time ](https://github.com/backstage/backstage/pull/4401#discussion_r588174750)that wasn't the case so thats why its a duplicate in the techdocs plugin, thanks for pointing that out, I'll update!